src/DetailsView/components/adhoc-tab-stops-test-view.tsx

```diff
@@ -103,6 +103,7 @@ export const AdhocTabStopsTestView = NamedFC<AdhocTabStopsTestViewProps>(
         );
 
         if (props.tabStoreData.isChanged) {
+            props.deps.detailsViewActionMessageCreator.targetPageChangedResetData();
```

Comment:
I think this makes sense as a quick fix, but I feel like it might be preferable to have whatever is responding to this message instead hook directly into whatever message sets `tabStoreData.isChanged`; it feels off for the data clearing behavior to rely on whether particular bits of UI are rendered

Reply:
This makes sense, it's tied into `Messages.Tab.ExistingTabUpdated` which is listened to from several stores. I made the change to hook directly into this message instead.